docs/installation/windows.md

```diff
@@ -346,7 +346,7 @@ uses. You can do this with
 [puttygen](http://www.chiark.greenend.org.uk/~sgtatham/putty/download.html):
 
 1. Open `puttygen.exe` and load ("File"->"Load" menu) the private key from
-  `%USERPROFILE%\.ssh\id_boot2docker`
+  `%USERPROFILE%.docker\machine\machines<name_of_your_machine>`
```

Comment:
I think this misses some slashes (after `%USERPROFILE%`, and before `<name_of_your_machine>`).
Can you change this to;

    %USERPROFILE%\.docker\machine\machines\<name_of_your_machine>\